src/Behat/Transformation/Transform.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Transformation;
+
+/**
+ * Represents an Attribute for a Transform transformation
+ */
+#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+final class Transform
+{
+    public function __construct(
+        public string $pattern = ''
```

Comment:
should we make it readonly as it is public ? I don't think we should allow mutating the attribute instance.

Reply:
I did not add `readonly` because I thought that was PHP 8.2. Now I realise that PHP 8.2 introduced readonly classes, readonly properties were available in PHP 8.1. It's hard to keep track of all these changes! 🤣 Updated